order/utils/mqtt.go

```diff
@@ -43,18 +44,21 @@ var connectLostHandler mqtt.ConnectionLostHandler = func(client mqtt.Client, err
 	fmt.Printf("Connect lost: %v", err)
 }
 
-func Publish(client mqtt.Client, topic string) {
-	num := 100
-	for i := 0; i < num; i++ {
-		text := fmt.Sprintf("Message %d de la %s", i, mqttUsername)
-		token := client.Publish(topic, 0, false, text)
-		token.Wait()
-		time.Sleep(time.Second)
-	}
+func Publish(payload interface{}, client mqtt.Client, topic string) {
+	fmt.Printf("\npublishing to topic: %v with payload: %v\n", topic, payload)
+	token := client.Publish(topic, 0, true, payload)
+	token.Wait()
+	//time.Sleep(time.Second)
 }
 
 func Subscribe(client mqtt.Client, topic string) {
 	token := client.Subscribe(topic, 1, nil)
 	token.Wait()
```

Comment:
Shouldn't there be some error handling involved here (after this line)? Something like: 
```
if token.Error() != nil
 do something with the error
```